src/app/tree-grid-load-on-demand/tree-grid-load-on-demand.sample.ts

```diff
@@ -21,7 +21,6 @@ export class MySummaryOperand extends IgxSummaryOperand {
 }
 
 @Component({
-    providers: [{ provide: IgxGridTransaction, useClass: IgxHierarchicalTransactionService }],
```

Comment:
delete also `IgxHierarchicalTransactionService` and `IgxGridTransaction` from the imports.